src/monattack.cpp

```diff
@@ -2859,11 +2861,28 @@ bool mattack::stare( monster *z )
         if( g->u.sees( *z ) ) {
             add_msg( m_bad, _( "The %s stares at you, and you shudder." ), z->name() );
         } else {
-            add_msg( m_bad, _( "You feel like you're being watched, it makes you sick." ) );
+            add_msg( m_bad, _( "You feel like you're being watched; it makes you sick." ) );
+        }
+       //Apply ?taint" to the target
```

Comment:
```suggestion
       //Apply "taint" to the target
```
BTW this is rather useless comment, the code below is easy to understand even without it.

Reply:
It was only added in case I added more to the code. I just didn't think to remove it.